sdk/eventgrid/eventgrid-namespaces/README.md

```diff
@@ -0,0 +1,181 @@
+# Azure Event Grid Namespaces client library for JavaScript
+
+[Azure Event Grid](https://azure.microsoft.com/services/event-grid/) is a cloud-based service that provides reliable event delivery at massive scale.
+
+Use the client library to Send events to Event Grid Namespaces 
+
+Key links:
+
+- [Source code](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/eventgrid/eventgrid/)
+- [Package (NPM)](https://www.npmjs.com/package/@azure/eventgrid-namespaces)
+- [API reference documentation](https://docs.microsoft.com/javascript/api/@azure/eventgrid/)
+- [Product documentation](https://docs.microsoft.com/azure/event-grid/)
+- [Samples](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/eventgrid/eventgrid/samples)
```

Comment:
are you going to update the links to point to the new package?

Reply:
Yes. Once the merge is complete. Else the Link checker fails the job